### PR TITLE
[EA] Improve page titles and descriptions (mainly for SEO purposes)

### PR DESCRIPTION
**Pull request by Will-Howard**
This is mainly adding descriptions to pages that didn't have them (and were just using the default description of "Effective Altruism research, discussion and community updates."). The specific wording has been discussed in [this doc](https://docs.google.com/document/d/1rH9tlliHVvc4K5D4Xrm2e2FLAQOFwJGtcM7yV_XdC8k/edit) (only accessible to CEA members), if you want to suggest changes to just the wording please suggest them in the doc (and maybe flag that you have done so here) so I can keep that as the source of truth.

There are a few other things in this PR:
 - Adding a `noindex` option to tags (so we can hide the "wiki" topic from search results, as people should go to /topics/all instead)
 - A fairly significant change to the `/community` (now `/groupsAndPeople` page). I have removed the bit above the banner because we want to remove the word "Connect" and I didn't think it was adding much to the page:
Screenshot 2023-06-21 at 10 50 01
Screenshot 2023-06-21 at 10 59 05

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204855954531077) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/community/modules/CommunityBanner.tsx [hunk 1]
```diff
@@ -5,14 +5,16 @@ import { createStyles } from '@material-ui/core/styles';
 const styles = createStyles((theme: ThemeType): JssStyles => ({
   bannerImg: {
     width: '100vw',
-    height: 200,
+    height: 220,
     backgroundImage: `linear-gradient(to right, ${theme.palette.primary.dark} 20%, transparent), url(https://res.cloudinary.com/cea/image/upload/c_fill,h_380,w_1600,q_auto,f_auto/236225045_2995791057331456_5749161116892625450_n.jpg.jpg)`,
     backgroundSize: 'cover',
     backgroundPosition: 'top right',
-    padding: 40,
+    padding: 50,
```
Comment (Will-Howard) [inline packages/lesswrong/components/community/modules/CommunityBanner.tsx:12]: Slightly adjusted the padding to make the page look more balanced without the extra space at the top. The only downside of this is that the page is now a bit short on large screens, but I think it's better to have more of the page devoted to functionality than to pad it out

File: packages/lesswrong/components/titles/PostsPageHeaderTitle.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const PostsPageHeaderTitle = ({isSubtitle, siteName, classes}: {
   });
   
   if (!post || loading) return null;
-  const titleString = `${post.title} - ${siteName}`
+  const titleString = `${post.title} — ${siteName}`
```
Comment (Will-Howard) [inline packages/lesswrong/components/titles/PostsPageHeaderTitle.tsx:23]: hyphen to em-dash

File: packages/lesswrong/lib/collections/tags/schema.ts [hunk 3]
```diff
@@ -630,6 +630,16 @@ const schema: SchemaType<DbTag> = {
     group: formGroups.advancedOptions,
     nullable: true,
   },
+  noindex: {
```
Comment (Will-Howard) [inline packages/lesswrong/lib/collections/tags/schema.ts:633]: `noindex` and `noIndex` are used about an equal amount in the codebase at the moment, I went with `noindex` for no particular reason

**Comment by agnesstenlund** [comment]
This looks great 👍 agree with removing the section on top

**Comment by s-cheng** [comment]
Sorry but I think I'm against removing the top section. I remember when Lotte and I did user interviews for this, people were often confused by what the point of this page was. I think this is worse on mobile - if you just load this page on a phone, without the top section, the portion above the fold tells you very little about what this page even is. (And it's missing the map, which is a big hint on desktop.)

Feel free to change any of the copy though, I'm sure it could be improved (like you can change the title and description to match the page metadata if you want)

**Review by oetherington — APPROVED** [review]
👍 Code checks out.

I don't have a particularly strong opinion but I'm mildly in favour of removing the top section. It definitely looks better without, and I think that if someone doesn't understand what that page is for then that sentence at the top probably won't help all that much. I'm guessing this page is up for a proper redesign at some point as well?

**Comment by Will-Howard** [comment]
@s-cheng 
> people were often confused by what the point of this page was

I feel like the name change will help with this?

**Comment by s-cheng** [comment]
I think changing the name helps a bit but it's still unclear what to do on this page when you arrive. Like, the primary audience of the page is people who are new to EA, and don't even know how the community is organized, so the term "groups" is pretty unclear to them. I'm sure the description could be improved, but the idea was to tell/encourage visitors that they can reach out and connect with these groups/people.

I also agree that this page will likely get redesigned at some point, but it seems unlikely to be prioritized so if we make a change now I think it will stick around for a long time. Sorry, that's a big factor in why I'm pushing back here.

(That said, I think it's reasonable for @agnesstenlund to be the decider - you can take my comments as just being input and still merge this in)

**Comment by agnesstenlund** [comment]
thanks for that context + pushback @s-cheng 🙏  ideally we'd show this to users again to be sure. Like Will I'm still leaning towards the name + rest of the page (the tabs with local + online groups, and the names of the groups) helping enough that the top bar doesn't add much–at least in it's current state–since the sub-header is also vague _"Effective altruism is a global community with thousands of members. Reach out to learn how you can have the most impact"._ (The second line doesn't really describe the real CTA on this page).

On the other hand, I was thinking that removing it would be inconsistent with the Events page:

Screenshot 2023-06-26 at 11 23 48

But thinking about it for longer I'm also realising the events page looks kind of odd with this section, and a better version likely only includes the sub-header (or nothing at all). So I'd still vote for removing it.

What I might consider doing though to help make it more clear what this page is about is changing "People" to "Members", if helpful. So "Groups & Members" instead of "Groups & people"

**Comment by agnesstenlund** [comment]
fair enough 👍

**Pull request merged**